tests/models/test_triplestoreconfig.py

```diff
@@ -58,10 +58,14 @@ def test_triplestoreconfig() -> None:
     }
 
     settings.expose_secrets = False
-    assert TripleStoreConfig(**config).json() == json.dumps(config_hidden)
+    assert TripleStoreConfig(**config).model_dump_json(
```

Comment:
consider making a comment why token, client_id and client_secret must be excluded

Reply:
I've updated this logic to again by default exclude these fields when serializing the `TripleStoreConfig` model - so any changes here can be reverted (except those related to changes to `json.dumps()`.